src/beamlime/kafka/message_adapter.py

```diff
@@ -185,6 +194,18 @@ def adapt(
         return replace(message, value=da00_to_scipp(message.value))
 
 
+class BeamlimeCommandsAdapter(MessageAdapter[KafkaMessage, Message[Any]]):
+    """
+    Adapts a Kafka message to a Beamlime command message.
+    """
+
+    def adapt(self, message: KafkaMessage) -> Message[Any]:
+        key = MessageKey(topic=message.topic(), source_name='')
+        legacy_key = message.key().decode('utf-8')  # See 286
+        value = message.value()
+        return Message(key=key, timestamp=0, value={'key': legacy_key, 'value': value})
```

Comment:
Kafka messages have timestamp (timestamp-type and actual-timestamp) so we can probably just use it, unless `0` has a specific meaning in this context.

https://docs.confluent.io/platform/current/clients/confluent-kafka-python/html/index.html#confluent_kafka.Message.timestamp

Reply:
Ah, is that why I got a tuple of two ints as a timestamp? I tried using it first but could not make sense of it.